src/Components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useProducts } from "../../Context/Product-Context/product-context";
+import "./productcard.css";
+export const ProductCard = () => {
+  const {productsList} = useProducts();
+  return (
+    <div className="all-products">
+      {productsList.map((item) => (
```

Comment:
You could add a ternary or logical and operator here like productsList && productsLis.map(.......

Reply:
Yes thats a good point , I'll make the changes, Thanks for suggesting that